### PR TITLE
command suggestions and parameters animation with native driver

### PR DESCRIPTION
**Pull request by rasom**
status: ready

### CODE REVIEW

File: src/status_im/ui/screens/chat/styles/animations.cljs [hunk 1]
```diff
@@ -13,17 +13,16 @@
 
 (defn expandable-container [anim-value bottom max-height]
```
Comment (jeluard) [inline src/status_im/ui/screens/chat/styles/animations.cljs]: Is bottom still used?

File: src/status_im/ui/screens/chat/input/animations/expandable.cljs [hunk 2]
```diff
@@ -9,21 +9,24 @@
...
+  (animation/start
+   (animation/spring anim-value {:toValue         -53
+                                 :friction        10
+                                 :tension         60
+                                 :useNativeDriver true})))
 
-(defview expandable-view [{:keys [key]} & elements]
-  (letsubs [anim-value         (animation/create-value 0)
+(defview expandable-view [_ & elements]
+  (letsubs [anim-value         (animation/create-value 104)
```
Comment (jeluard) [inline src/status_im/ui/screens/chat/input/animations/expandable.cljs]: Can we infer or comment those magic numbers?

**Review by jeluard — APPROVED** [review]

File: src/status_im/ui/screens/chat/styles/animations.cljs (hunk 1 above)
Comment (rasom) [reply src/status_im/ui/screens/chat/styles/animations.cljs]: nope it's not, fixed

File: src/status_im/ui/screens/chat/input/animations/expandable.cljs (hunk 2 above)
Comment (rasom) [reply src/status_im/ui/screens/chat/input/animations/expandable.cljs]: added a comment

**Review by mandrigin — APPROVED** [review]

**Comment by Serhy** [comment]
@rasom, threre are regressions with current changes:

On **iOS and Android**:
1) List of assets that appears for `/send` / `/request` commands is not scrollable. 
2) List of assets has different max-size: on Android whole list of assets revealed disregard top border max limit; on iOS list of assets max height seems applied but it's different that what is in develop
(please see screenshots below for comparison with develop)
Screenshot 2019-06-26 at 13 14 17
Screenshot 2019-06-26 at 13 14 24

3) On **iOS only**:
command suggestions does not appear when tap upon `[/]` button or start typing message with `/`. If type dirrectly `/send` or `/request` in message box then assets appears but list height is different (see above items).

Screenshot 2019-06-26 at 13 06 07

**Comment by rasom** [comment]
@Serhy both issues should be fixed now

**Comment by Serhy** [comment]
Looks good now (tested Androids 6.0.1, 8.0, 8.1 and iOS 12)
Thanks a lot, @rasom !

**Pull request merged**